src/App.js

```diff
@@ -80,7 +82,13 @@ class Root extends React.Component {
   render() {
     if (this.props.isVersionAllowed === undefined) {
       // Waiting for version
-      return null;
+      return (
+        <Router>
+          <>
+            <Navigation />
```

Comment:
What happens if the user clicks the navigation buttons while the app is loading?

Reply:
The page just change, but the loading component remains. I've added a video showing it on PR description